### ISSUE TITLE
[Feature] Lock status API

### ISSUE THREAD

**Issue by html5cat** [smartcar/demo#6]
First of all, holy cow, I'm on a plane playing with car api sample app ✈️❤️  
I think it would be awesome to have a state of locks endpoint – right now it seems that I can only send request to lock or unlock. 

Thank you!

**Comment by esonmez** [comment]
@html5cat thank you so much for trying it out!! Definitely agreed on car state  :+1:  :+1:  I've added this to our feature request list for our team.

p.s. talking to cars on the ground from 30,000 feet may be the subject of a future blog post... let us know if you want to be featured 😉

**Comment by html5cat** [comment]
Too late, already writing it 😸Unless you want to write it, in which case definitely would like to be featured. (if we don't fall off the sky on this bumpy AA 2305)

**Comment by html5cat** [comment]
"Did I lock the car?" could be an awesome sample app that is also very useful, possibly viral.

**Comment by html5cat** [comment]
Yup, pretty accurate. Thanks to Mathilde for reaching out few hours ago ;)
